src/main/pages/che-7/installation-guide/proc_upgrading-che-using-operatorhub.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// upgrading-{prod-id-short}
+
+[id="upgrading-{prod-id-short}-using-operatorhub_{context}"]
+= Upgrading {prod-short} using OperatorHub
+
+This section describes how to upgrade from previous minor version using the Operator from OperatorHub in the OpenShift web console.
+
+.Prerequisites
+
+* An administrator account on an OpenShift instance.
+
+* An instance of a previous minor version of {prod-short}, installed using an Operator from OperatorHub on the same instance of OpenShift.
+
+.Procedure
+
+. Open the OpenShift web console.
+
+. Navigate to the *Operators* -> *Installed Operators* section.
+
+. Click *{prod}* in the list of installed operators.
+
+. Navigate to the *Subscription* tab and enable the following options:
```

Comment:
enable > specify ?

Not sure about this one, need to check how it looks in the GUI

Reply:
GUI may change very quickly :). Today it's a on/off slider.